roles/cifmw_ceph_client/README.md

```diff
@@ -51,4 +70,36 @@ metadata:
   namespace: openstack
 type: Opaque
 ```
-The user should then run `kubectl create -f /tmp/k8s_ceph_secret.yml`.
+The user could then run `kubectl create -f /tmp/k8s_ceph_secret.yml`.
+
+Alternatively, a copy of
+https://github.com/openstack-k8s-operators/architecture
+may be cloned to `/home/zuul` and the role may be called like this:
+
+```yaml
+- name: Render Ceph client configuration
+  hosts: localhost
+  gather_facts: false
+  vars:
+    cifmw_ceph_client_vars: /tmp/ceph_client.yml
```

Comment:
I don't think this one is documented? Maybe you wanted to point to `cifmw_ceph_client_values_post_ceph_path_dst`?

Reply:
All variables used in the example playbook between lines 79 and 90 of the README are documented under `## Parameters` between lines 16 and 41.

Reply:
Because `cifmw_ceph_client_values_post_ceph_path_dst` and `cifmw_ceph_client_service_values_post_ceph_path_dst` have default values we don't need to pass them here.

Reply:
oh, ok, I didn't see it was present in the masked part of the README - sorry!